lib/evss/disability_compensation_auth_headers.rb

```diff
@@ -30,14 +30,14 @@ def eauth_json
     end
 
     def gender
-      case @user.gender || @user.mpi&.profile&.gender
+      case @user.gender || @user.gender
```

Comment:
These seems like an opportunity to standardize what "gender" is to us? Is it the word or just the letter? I don't think we should be using gender here this way though.

Reply:
Haha, whoops!